### DISCUSSION TITLE
[Request] X68000 rom selection from main games menu - update core

### DISCUSSION THREAD

**Discussion by paladinperry** [OnionUI/Onion#1155] [General]
Thanks for the hard work guys I'm loving onion on my Miyoo Mini +

Is there any chance the x68000 can be added to the main games list menu with its own sub folder just like SNES/PSX?Megadrive etc ?
also does the core need an update ? I've not been able to get my x68000 roms to work from expert menu yet either

One last thing. when does the next Beta or Alpha release drop ?

Thanks again :)

**Comment by Phanturian** [comment]
The X68000 system is more complicated so I understand why it's in the Expert menu. It's not as easy as dropping roms in the folder and playing. I checked just now and a pack I made for X68000 over a year ago works fine in the Expert menu. I apologize if anything I'm about to suggest is obvious to you, but I will provide info in case anyone else wants to give it a try.

1. Install Sharp-X68000 (PX68k) from the EXPERT section of the Package Manager. Go to Apps>Tweaks>Appearance>Show expert mode and turn it ON. You now have the expert section on the main menu.

2.  Make sure you have the correct bios: iplrom.dat, cgrom.dat, iplrom30.dat (optional), iplromco.dat (optional), iplromxv.dat (optional) (need to be placed in a folder named keropi within the BIOS folder). I suggest including all the bios listed.

3. Use roms with the extensions .hdf and .hdm (the wiki says to use .dim and .m3u but I haven’t tested these). These examples have been tested:
   
Akumajou Dracula.hdf
Cotton (Disk A).hdm
Cotton (Disk B).hdm

4. You can use the tutorial picture below to get Cotton running. For Akumajou Dracula, just make sure to choose option 1 of 3 when it boots, otherwise the game won’t have music.

![X68000 Instructions](https://github.com/OnionUI/Onion/assets/135469050/9937500b-a765-4e84-b3f2-e5b7ce70df5f)

**Reply by paladinperry** [reply]
Thank you so much mate I responded below

**Comment by paladinperry** [comment]
Cheers mate, I really appreciate you going to the effort to reply with a tutorial for me.

Yeah I had the expert menu on homepage and the px68k core etc already, i've tried a few different formats as I have about 20gb of roms with different extensions.
I didn't know about loading in the floppy disk drive 2 etc, so i've learned something new.

I know x68000 is a pain in the arse, I didn't mean it to sound like a simple task, sorry if it sounded that way.
All the systems I enjoy are a nightmare (x68000/saturn/fm-towns) I was just hoping one day it could have its own neat little
icon picture with all the other systems ha.

The OS/front end runs so good on the miyoo I love it.

Thank you so much to you and the other devs, it's great being able to hoard my favourite roms in such a tiny good looking package
